ui/src/components/ConnectorGrid/index.tsx

```
@@ -1,19 +1,37 @@
 export interface ConnectorGridProps {
-    list: Array<object>;
+    // list: Array<object>;
+    active: ActiveConnectorState;
+    setActive: React.Dispatch<React.SetStateAction<{ name: string; icon: string }>>;
+    handleOnClickGrid: () => void;
 }
 
-const ConnectorGrid: React.FC<ConnectorGridProps> = ({ list }) => {
-    console.log(list);
+const ConnectorGrid: React.FC<ConnectorGridProps> = ({ active, setActive, handleOnClickGrid }) => {
+    const [active, setActive] = useState<ActiveConnectorState>({ name: '', icon: '' });
```

Biome

[error] Shouldn't redeclare 'active'. Consider to delete it or rename it.

'active' is defined here:


(lint/suspicious/noRedeclare)